po/nl_NL.po

```diff
@@ -0,0 +1,464 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
```

Comment:
Mind fixing the copyright header please? :) 
something like this would work
```
# Copyright (C) 2019
# This file is distributed under the same license as the PACKAGE package.
#
# your name <your email>, 2019.
```

Reply:
Just fixed it :)